views/lang/latin/verb/irregular/latin-verb-irregular-view.js

```diff
@@ -67,7 +67,11 @@ export default class LatinVerbIrregularView extends LatinVerbIrregularBaseView {
    * A list of constructors of linked views.
    * @return {View[]}
    */
-  static get linkedViewConstructors () {
-    return [LatinVerbParicipleIrregularView, LatinVerbSupineIrregularView]
+  static linkedViewConstructors (homonym) {
+    const views = [LatinVerbParicipleIrregularView]
```

Comment:
Maybe according to our rules we've agreed upon before about 'let' and 'const' we should make this `let`? The reason is we're changing it later with the `push()`. We have to add `// esling-disable-line prefer-const` too to prevent linter from changing this line back. What do you think?

Reply:
yes I agree, thanks for catching that. I will fix it.